docs/how-to/deploy_sdcore_production.md

```diff
@@ -0,0 +1,328 @@
+```
+
+Install the vfioveth CNI:
+
+```console
+sudo wget -O /opt/cni/bin/vfioveth https://raw.githubusercontent.com/opencord/omec-cni/master/vfioveth
+sudo chmod +x /opt/cni/bin/vfioveth
+```
+
+Create the `ipaddresspools.yaml` manifest for the static IP address pools for MetalLB,
+using the 4 not configured static IP address from the management network, to update the
+following content:
+
+```yaml
+---
+apiVersion: metallb.io/v1beta1
+kind: IPAddressPool
+metadata:
+  name: lb-address-cos
+  namespace: metallb-system
+spec:
+  addresses:
+  - 10.201.0.3/32
+  avoidBuggyIPs: false
+  serviceAllocation:
+    namespaces:
+      - cos-lite
+
+---
+apiVersion: metallb.io/v1beta1
+kind: IPAddressPool
+metadata:
+  name: lb-address-nms
+  namespace: metallb-system
+spec:
+  addresses:
+  - 10.201.0.4/32
+  avoidBuggyIPs: false
+  serviceAllocation:
+    namespaces:
+      - control-plane
+    serviceSelectors:
+      - matchExpressions:
+        - {key: "app.juju.is/created-by", operator: In, values: [traefik]}
+
+---
+apiVersion: metallb.io/v1beta1
+kind: IPAddressPool
+metadata:
+  name: lb-address-amf
+  namespace: metallb-system
+spec:
+  addresses:
+  - 10.201.0.5/32
+  avoidBuggyIPs: false
+  serviceAllocation:
+    namespaces:
+      - control-plane
+    serviceSelectors:
+      - matchExpressions:
+        - {key: "app.juju.is/created-by", operator: In, values: [amf]}
+
+---
+apiVersion: metallb.io/v1beta1
+kind: IPAddressPool
+metadata:
+  name: lb-address-upf
+  namespace: metallb-system
+spec:
+  addresses:
+  - 10.201.0.6/32
+  avoidBuggyIPs: false
+  serviceAllocation:
+    namespaces:
+      - user-plane
+```
+
+Apply the manifest:
+
+```console
+sudo k8s kubectl apply -f ipaddresspools.yaml
+```
+
+Extract the Kubernetes configuration to a file:
+
+```console
+sudo k8s config > sdcore_k8s_config
+```
+
+Transfer the resulting file to the machine used for installation.
+
+Reboot:
+
+```console
+sudo reboot
+```
+
+## Bootstrap a Juju controller
+
+The remaining steps need to be run from the installation machine.
+
+Add the Kubernetes cluster to the Juju client and bootstrap the controller:
+
+```console
+export KUBECONFIG=/path/to/sdcore_k8s_config
+juju add-k8s sdcore_k8s --cluster_name=k8s --client
+juju bootstrap --config controller-service-type=loadbalancer sdcore_k8s
+```
+
+## Deploy SD-Core
+
+Create a Terraform module for your deployment:
+
+```console
+mkdir terraform
+cd terraform
+```
+
+Create a `main.tf` file with the following content, updating the values for your deployment:
+
+```terraform
+module "sdcore-production" {
+  source = "git::https://github.com/canonical/charmed-aether-sd-core//production?ref=feat-prod-deployment"
+
+  amf_ip = "10.201.0.12"
+  amf_hostname = "amf.example.com"
+  gnb_subnet = "10.204.0.0/24"
+  nms_domainname = "sdcore.example.com"
+  upf_access_gateway_ip = "10.202.0.1"
+  upf_access_ip = "10.202.0.10/24"
+  upf_access_mac = "a1:b2:c3:d4:e5:f6"
```

Comment:
Should the users have noted the mac addresses before rebooting? If yes we should probably tell them.

Reply:
Good point, I think I forgot to add that.

Reply:
I added a step to record the MAC addresses.